addons/stock/views/product_views.xml

```diff
@@ -303,7 +301,12 @@
                         <button type="object"
                             name="action_open_quants"
                             attrs="{'invisible':[('type', '!=', 'product')]}"
-                            class="oe_stat_button" icon="fa-building-o">
+                            class="oe_stat_button" icon="fa-building-o"
+                            context="{
+                                'single_product': product_variant_count == 1,
+                                'quantity_available_locations_domain': ('internal','transit'),
```

Comment:
we need to double check why this context key is needed

Reply:
It seems it's make no difference, I think I misunderstood what it does when I added that. I remove it.